pubnub-gson/pubnub-gson-impl/src/main/java/com/pubnub/internal/java/v2/PNConfigurationImpl.kt

```diff
@@ -81,12 +84,26 @@ class PNConfigurationImpl(
         const val CONNECT_TIMEOUT = 5
     }
 
+    // this method is used from cryptoModuleWithLogConfig using reflection
+    fun getCryptoModuleWithLogConfig(logConfig: LogConfig): CryptoModule? {
```

Comment:
What's the purpose of this method? Is it to inject `logConfig` property to `CryptoModule`?

Reply:
Why customer creates cryptoModule it create it without logConfig. It is hidden from them. Then we need to add logConfig so that cryptoModule operations can be logged.